docs/deployment/rds-s3/kustomization.yaml

```diff
@@ -1,63 +1,13 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+bases:
+- ./base
 resources:
-# Cert-Manager
-- ../../../upstream/common/cert-manager/cert-manager/base
-- ../../../upstream/common/cert-manager/kubeflow-issuer/base
-# Istio
-- ../../../upstream/common/istio-1-9/istio-crds/base
-- ../../../upstream/common/istio-1-9/istio-namespace/base
-- ../../../upstream/common/istio-1-9/istio-install/base
-# OIDC Authservice
-- ../../../upstream/common/oidc-authservice/base
-# Dex
-- ../../../upstream/common/dex/overlays/istio
-# KNative
-- ../../../upstream/common/knative/knative-serving/overlays/gateways
-- ../../../upstream/common/knative/knative-eventing/base
-- ../../../upstream/common/istio-1-9/cluster-local-gateway/base
-# Kubeflow namespace
-- ../../../upstream/common/kubeflow-namespace/base
-# Kubeflow Roles
-- ../../../upstream/common/kubeflow-roles/base
-# Kubeflow Istio Resources
-- ../../../upstream/common/istio-1-9/kubeflow-istio-resources/base
-
-
-# KFServing
-- ../../../upstream/apps/kfserving/upstream/overlays/kubeflow
-# Central Dashboard
-- ../../../upstream/apps/centraldashboard/upstream/overlays/istio
-# Admission Webhook
-- ../../../upstream/apps/admission-webhook/upstream/overlays/cert-manager
-# Notebook Controller
-- ../../../upstream/apps/jupyter/notebook-controller/upstream/overlays/kubeflow
-# Jupyter Web App
-- ../../../awsconfigs/apps/jupyter-web-app
-# Profiles + KFAM
-- ../../../upstream/apps/profiles/upstream/overlays/kubeflow
-# Volumes Web App
-- ../../../upstream/apps/volumes-web-app/upstream/overlays/istio
-# Tensorboards Web App
-- ../../../upstream/apps/tensorboard/tensorboard-controller/upstream/overlays/kubeflow
-# Tensorboard Controller
-- ../../../upstream/apps/tensorboard/tensorboards-web-app/upstream/overlays/istio
-# Training Operator
-- ../../../upstream/apps/training-operator/upstream/overlays/kubeflow
-# MPI Operator
-- ../../../upstream/apps/mpi-job/upstream/overlays/kubeflow
-# AWS Telemetry - This is an optional component. See usage tracking documentation for more information
-- ../../../awsconfigs/common/aws-telemetry
-
-# User namespace
-- ../../../upstream/common/user-namespace/base
-
-
 # Configured for AWS RDS and AWS S3
 
 # AWS Secret Manager
-- ../../../awsconfigs/common/aws-secrets-manager/base
+- ../../../awsconfigs/common/aws-secrets-manager
```

Comment:
this change need to goto:
- https://github.com/awslabs/kubeflow-manifests/blob/main/docs/deployment/cognito-rds-s3/kustomization.yaml#L58
- https://github.com/awslabs/kubeflow-manifests/blob/main/tests/e2e/utils/notebooks/setup_secrets_access.py#L11

Reply:
done